x-pack/plugins/lists/server/services/extension_points/extension_point_storage_client.test.ts

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { loggerMock } from '@kbn/logging/mocks';
+
+import { CreateExceptionListItemOptions } from '../exception_lists/exception_list_client_types';
+import { getCreateExceptionListItemOptionsMock } from '../exception_lists/exception_list_client.mock';
+import { DataValidationError } from '../exception_lists/utils/errors';
+
+import { ExtensionPointStorageClientInterface } from './extension_point_storage_client';
+import { ExtensionPointError } from './errors';
+import { ExtensionPointStorageInterface } from './extension_point_storage';
+import {
+  ExceptionListPreUpdateItemServerExtension,
+  ExceptionsListPreCreateItemServerExtension,
+  ExtensionPoint,
+} from './types';
+import { createExtensionPointStorageMock } from './extension_point_storage.mock';
+
+describe('When using the ExtensionPointStorageClient', () => {
+  let storageClient: ExtensionPointStorageClientInterface;
+  let logger: ReturnType<typeof loggerMock.create>;
+  let storageService: ExtensionPointStorageInterface;
+  let preCreateExtensionPointMock1: jest.Mocked<ExceptionsListPreCreateItemServerExtension>;
+  let extensionPointsMocks: Array<jest.Mocked<ExtensionPoint>>;
+  let callbackRunLog: string;
+
+  const addAllExtensionPoints = (): void => {
+    extensionPointsMocks.forEach((extensionPoint) => storageService.add(extensionPoint));
+  };
+
+  beforeEach(() => {
+    const storageContext = createExtensionPointStorageMock();
+
+    callbackRunLog = '';
+    // eslint-disable-next-line prefer-destructuring
```

Comment:
You can remove this by using destructoring like so:

```ts
({ logger, extensionPointStorage: storageService } = storageContext);
storageService.clear();
```

Personally I wouldn't rename `extensionPointStorage` to be `storageService` but rather keep the same name with the `let` variable above so you could do:

```ts
({ logger, extensionPointStorage } = storageContext);
extensionPointStorage.clear();
```

But even with renaming at least we don't add additional linter disablements to the code.

Reply:
👍 
Will make the changes. Thanks.